ppr-ui/src/composables/mhrRegistration/home-owner.ts

```diff
@@ -62,7 +62,7 @@ export function useHomeOwners (isPerson: boolean = false, isEditMode: boolean =
     if (showGroups.value) {
       // At leas one group showing with one or more owners
       return HomeTenancyTypes.COMMON
-    } else if (numOfOwners === 1) {
+    } else if (numOfOwners === 1 && getMhrRegistrationHomeOwners.value[0].address !== undefined) {
```

Comment:
If we are looking for just proof the property exists or is initialized, does `!!getMhrRegistrationHomeOwners.value[0].address` work?

Also, should we optionally chain `getMhrRegistrationHomeOwners.value[0]` or is this always initialized?